### PR TITLE
added test to function getOverlayConfig from zedroutertypes.go

### PR DESCRIPTION
**Pull request by archishou**
Signed-off-by: Archishmaan Peyyety <archishou@gmail.com>

### CODE REVIEW

**Comment by archishou** [comment]
I'm not sure why build is failing. The tests run fine on my machine and I tried running both go fmt and go vet. Anything I'm missing?

File: pkg/pillar/types/zedroutertypes_test.go [hunk 1]
```diff
@@ -4,10 +4,24 @@
...
 
 import (
+	"github.com/satori/go.uuid"
 	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
+var underlayUUID = uuid.UUID{0x6b, 0xa7, 0xb8, 0x10, 0x9d, 0xad, 0x11, 0xd1,
+	0x80, 0xb4, 0x00, 0xc0, 0x4f, 0xd4, 0x30, 0xc8}
+var overlayUUID = uuid.UUID{0x6b, 0xa7, 0xb8, 0x10, 0x9d, 0xad, 0x11, 0xd1,
```
Comment (eriknordmark) [inline pkg/pillar/types/zedroutertypes_test.go]: Shouldn't they be different?

**Comment by rvs** [comment]
Hey @archishou it is clear from the log that go fmt is failing on: ./types/zedroutertypes_test.go

I'm pretty sure you can reproduce the same on your laptop by simply running:
```
make pkg/pillar
```

Makes sense?

**Comment by archishou** [comment]
Thats what I thought as well, but running go fmt in the /types directory didn't result in any changes.

File: pkg/pillar/types/zedroutertypes_test.go (hunk 1 above)
Comment (archishou) [reply pkg/pillar/types/zedroutertypes_test.go]: Yep, I must have copied the wrong thing. Let me fix that.

**Comment by rvs** [comment]
The difference @archishou being that when you run go fmt on your laptop you're using a different setup of Go compared to what we're using in a container that builds pillar.

That's why we have containers to begin with -- so we don't depend on how individual developer's laptops are setup.

**Comment by archishou** [comment]
Looks like everything passed, let me know if there is anything else.

File: pkg/pillar/types/zedroutertypes_test.go (hunk 1 above)
Comment (rvs) [reply pkg/pillar/types/zedroutertypes_test.go]: I'm confused. Wasn't this part supposed to be fixed @archishou ?

Comment (archishou) [reply pkg/pillar/types/zedroutertypes_test.go]: I did: https://github.com/lf-edge/eve/pull/108/commits/8c9cd9a065f6a859db98677646207d0642e0eb6c

**Comment by archishou** [comment]
Is there anything pending?

**Pull request merged**